src/NavBar/index.ts

```diff
@@ -1,2 +1,3 @@
 import NavBar from './NavBar';
 export default NavBar;
+export { NavBarProps } from './NavBar';
```

Comment:
I think is better write as
```js
import NavBar, { NavBarProps } from './NavBar';
export default NavBar;
export { NavaBarProps };
```

but it's just a personal opinion (and for consistency with other components)